### DISCUSSION TITLE
Second target.

### DISCUSSION THREAD

**Discussion by ozgurerdogan** [psy0rz/zfs_autobackup#188] [General]
I have one main backup that pull from other server. But now I want to store backups to a second backup server. So I need two targets. Whats is the best way for it? 
Two crons?

**Comment by psy0rz** [comment]
Yes two seperate crons. If they both use the same schedule and keep-settings, you can use the same backup name.

Otherwise use two separate backup names, this way you can make as many individual backups as you need, without conflicts.

**Comment by ozgurerdogan** [comment]
I want to pull backups to server-a first and copy those backups to server-b with higher keep-target value (than server-a).

**Comment by psy0rz** [comment]
thats also no problem, just make a seperate pull backup on server-a. (or push from server-b)

you can use the same backup names throughout, but make sure that the second job to server-b, doesnt destroy the common snaphots that are on server-a. to be use just use --keep-source=10000 for that second backup, so it never destroys snapshots on a.
